.github/workflows/regression-report.yml

```diff
@@ -636,7 +665,9 @@ jobs:
           ./.ci/append-to-github-output.sh "message" "$(cat .ci-temp/message)"
 
       - name: Comment PR
-        uses: checkstyle/contribution/comment-action@master
+        uses: peter-evans/create-or-update-comment@v3
```

Comment:
This is unrelated issue, 
Ok, we tested it in this PR, lets now move this change to separate PR with linkage to proper issue and merge it quickly.

Reply:
okay, the issue i had created aimed at removing contribution entirely, wouldn't this qualify a part of the issue?

Reply:
the bigger you make PR, the more complicated to merge it, it is always better to split something to separate PR and reference the same issue.

Reply:
removed, done.